packages/web-pkg/src/components/SideBar/FileSideBar.vue

```diff
@@ -100,7 +100,7 @@ export default defineComponent({
         return {
           root: props.space,
           parent: null,
-          items: unref(currentFolder)?.id ? [unref(currentFolder)] : []
```

Comment:
@kulmann This is important! I noticed that while opening 'spaces' or 'trash' two panels were rendered, first the details panel (not visible because it changes super fast) after that, the no selection panel.
The details panel should have never been rendered at all!
This happens because the currentFolder was actually set in Trash or Spaces (set to  the admin/user folder, if coming from personal) Leading to multiple issues.
I disabled showing the currentFolder for now, but we need to find a solution for that

Reply:
I tried to implement a real fix here https://github.com/owncloud/web/pull/10624/commits/27ae611b626b44a6671d2913946a73e3527b15c2

We need to reset the folder if we change to another context eg Personal to Trash and so on

Reply:
I currently see two ways to clean this up:

1. call `clearCurrentFolder` on the `resources` pinia store when unmounting the `GenericSpace` or `GenericTrash` view in the files app
2. call `clearCurrentFolder` on the `resources` pinia store before loading data for the trash overview or project spaces overview.

Reply:
Actually, the place where you applied a fix is fine as well. That whole code block seems to be a workaround for improper improper unmounting anyway, so doesn't hurt to get more specific in there. 😓